src/current/_includes/common/licensing-faqs.md

```diff
@@ -0,0 +1,148 @@
+1. [DB console]({% link {{ page.version.version }}/ui-overview.md %}): Several [license expiration messages]({% link {{ page.version.version }}/ui-overview.md %}#license-expiration-message) may be displayed, depending on the status of your cluster's license.
+
+## Renew an expired license
+
+To renew an expired **Enterprise** license, <a href="mailto:sales@cockroachlabs.com">contact Sales</a> and then [set](#set-a-license) the new license.
+
+To renew an expired **Enterprise Free** license, follow the steps in [Obtain a license](#obtain-a-license).
+
+## FAQs
+
+### Can I host CockroachDB as a service for internal use at my organization?
+
+Yes, employees and contractors can use your internal CockroachDB instance as a service, but no people outside of your organization will be able to use it without purchasing a license. Use of Enterprise features will always require a license.
+
+### What constitutes hosting CockroachDB as a service?
+
+Hosting CockroachDB as a service means creating an offering that allows third parties (other than your employees and contractors) to operate a database. Specifically, third parties cannot modify table schemas.
+
+<a name="throttling"></a>
+
+### What is throttling and how does it work?
+
+When a cluster is being throttled, the number of concurrent open [SQL transactions]({% link {{ page.version.version }}/transactions.md %}) is limited to 5.
+
+This will only happen in the following cases:
+
+- The cluster is not following telemetry requirements.
+    - There is a 7 day grace period for new **Enterprise Free** and **Enterprise Trial** clusters to start sending telemetry.
+- The cluster has an expired [license key](#obtain-a-license); depending on the type of expired license, the cluster will be throttled after the following time periods:
+    - **Enterprise**: Never throttles
+    - **Enterprise Free**: Throttles 30 days after expiration
+    - **Enterprise Trial**: Throttles 7 days after expiration
+    - No license: Throttles 7 days after cluster initialization
+
+Single node clusters for development use are [not throttled](#single-node-clusters).
+
+### Can I use CockroachDB for academic research?
+
+Cockroach Labs encourages non-commercial academic research involving CockroachDB. For such projects, please [contact us][support] to discuss a possible licensing arrangement.
+
+<a name="single-node-clusters"></a>
+
+### Do I need a license key for running a single node cluster?
+
+No license key is required for developers running [single-node clusters]({% link {{ page.version.version }}/cockroach-start-single-node.md %}) using the following [`cockroach` commands]({% link {{ page.version.version }}/cockroach-commands.md %}):
+
+- [`cockroach start-single-node`]({% link {{ page.version.version }}/cockroach-start-single-node.md %})
+- [`cockroach demo`]({% link {{ page.version.version }}/cockroach-demo.md %})
+
+Single node clusters are not [throttled](#throttling).
```

Comment:
might be good to add that you should only be using clusters without a license key for development purposes (not prod).

Reply:
added the same new blurb as above to the end of this section:

In general, self-hosted clusters without a license key are restricted to development purposes only. Development purposes includes non-production use in an internal development environment for design, prototype, or development purposes only. This restriction is enforced through [license terms and conditions][csl].